Assets/MixedRealityToolkit-SDK/StandardAssets/Controllers/Hands/Animations/l_hand_cap_touch_index_mask_anim.fbx.meta

```diff
@@ -0,0 +1,202 @@
+fileFormatVersion: 2
+guid: 8f30ac6cbbbe39d43886ad7626f7e2b2
+timeCreated: 1438879068
+licenseType: Pro
+ModelImporter:
+  serializedVersion: 18
+  fileIDToRecycleName:
+    100000: hands:b_l_grip
+    100002: hands:b_l_hand
+    100004: hands:b_l_hand_ignore
+    100006: hands:b_l_index1
+    100008: hands:b_l_index2
+    100010: hands:b_l_index3
+    100012: hands:b_l_index_ignore
+    100014: hands:b_l_middle1
+    100016: hands:b_l_middle2
+    100018: hands:b_l_middle3
+    100020: hands:b_l_middle_ignore
+    100022: hands:b_l_pinky0
+    100024: hands:b_l_pinky1
+    100026: hands:b_l_pinky2
+    100028: hands:b_l_pinky3
+    100030: hands:b_l_pinky_ignore
+    100032: hands:b_l_ring1
+    100034: hands:b_l_ring2
+    100036: hands:b_l_ring3
+    100038: hands:b_l_ring_ignore
+    100040: hands:b_l_thumb1
+    100042: hands:b_l_thumb2
+    100044: hands:b_l_thumb3
+    100046: hands:b_l_thumb_ignore
+    100048: hands:l_hand_world
+    100050: //RootNode
+    400000: hands:b_l_grip
+    400002: hands:b_l_hand
+    400004: hands:b_l_hand_ignore
+    400006: hands:b_l_index1
+    400008: hands:b_l_index2
+    400010: hands:b_l_index3
+    400012: hands:b_l_index_ignore
+    400014: hands:b_l_middle1
+    400016: hands:b_l_middle2
+    400018: hands:b_l_middle3
+    400020: hands:b_l_middle_ignore
+    400022: hands:b_l_pinky0
+    400024: hands:b_l_pinky1
+    400026: hands:b_l_pinky2
+    400028: hands:b_l_pinky3
+    400030: hands:b_l_pinky_ignore
+    400032: hands:b_l_ring1
+    400034: hands:b_l_ring2
+    400036: hands:b_l_ring3
+    400038: hands:b_l_ring_ignore
+    400040: hands:b_l_thumb1
+    400042: hands:b_l_thumb2
+    400044: hands:b_l_thumb3
+    400046: hands:b_l_thumb_ignore
+    400048: hands:l_hand_world
+    400050: //RootNode
+    7400000: l_hand_cap_touch_index_mask
+    9500000: //RootNode
+  materials:
+    importMaterials: 1
```

Comment:
Turn off all the material exports for every one of these fbx models please.

Reply:
Done, although it's not showing?